pkg/api/v1beta1/types.go

```diff
@@ -122,6 +122,8 @@ type PersistentVolumeSource struct {
 	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
 	// kubelet's host machine and then exposed to the pod.
 	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"persistentDisk" description:"GCE disk resource provisioned by an admin"`
+	// NFS represents an NFS mount on the host that persists beyond a pod's lifetime
+	NFS *NFSVolumeSource `json:"nfs" description:"NFS volume that will be mounted in the host machine "`
```

Comment:
We do want NFS as a real persistent volume.  I can send it along as a new PR for a quick merge, if you prefer that.  Then I can use that in my tests or I can change my tests to use HostPath.  I don't need NFS specifically for access modes that differ from GCE.  

I have no preference.